src/components/Layouts/SinglePage.tsx

```diff
@@ -83,6 +88,15 @@ const Layout: React.FC<Props> = ({ children }: Props) => {
                         >
                             Analysis
                         </Button>
+                        <Button
+                            variant="outlined"
+                            size="small"
+                            startIcon={<Icon baseClassName="icons">build</Icon>}
+                            onClick={openPreferencesDialog}
+                            sx={{ color: 'white' }}
+                        >
+                            Unit Preferences
```

Comment:
Let's rename this to "Preferences" and move it to the right of the API button. We might add more preferences to this later.

Reply:
Yup will do that.